js/src/common/utils/extractText.js

```diff
@@ -8,7 +8,7 @@ export default function extractText(vdom) {
   if (vdom instanceof Array) {
     return vdom.map((element) => extractText(element)).join('');
   } else if (typeof vdom === 'object' && vdom !== null) {
-    return extractText(vdom.children);
+    return vdom.children ? extractText(vdom.children) : vdom.text;
```

Comment:
Just for my understanding: Does Mithril VDOM now have a special case for text-only nodes?

Reply:
Yes. If it's text only, it'll be found in `vnode.text` as opposed to being an array of a string in `vnode.children` (I think that's what it previously was, anyway)

Reply:
Which is very annoying... But it is what it is, I suppose. I think we should just document when we need to hack around it (like in UsersSearchSource.

Reply:
Is the `else` case still relevant then? I would assume that can be removed then?

Reply:
No, because sometimes it'll still put text in a text vnode, which still uses children. For instance, if you run `m('p', [m('a', 'Hello!'), 'world']);` in browser console.